bin/propolis-standalone/src/main.rs

```diff
@@ -286,7 +286,29 @@ impl Instance {
         let state = &mut *state_guard;
         let machine = state.machine.as_ref().unwrap();
 
-        for vcpu in machine.vcpus.iter().map(Arc::clone) {
+        let bind_cpus = match this.0.config.main.cpu_binding {
+            Some(config::BindingStrategy::FromLast) => {
+                let mut bind_cpus = vec![None; machine.vcpus.len()];
+                let total_cpus =
+                    pbind::online_cpus().expect("can get processor count");
+                let vcpu_count: i32 =
+                    machine.vcpus.len().try_into().expect("<2^31 vCPUs");
```

Comment:
nit, considered not that big a deal: this is an error in the user's config file, rather than e.g. a syscall we expect should always work not working. can we make that a little clearer perhaps?